From 0d394eec9c00943eb3a5acfb1a75037cfb67aa38 Mon Sep 17 00:00:00 2001
From: Vadim Zeitlin <vadim@wxwidgets.org>
Date: Sat, 21 Oct 2017 17:42:30 +0200
Subject: [PATCH] Update to bakefile 0.2.11 and rebake everything

The main/only change in this version is the removal of automatic rules
using old Carbon Rez/DeRez/SetFile tools.
---
 Makefile.in                                      | 20 +++-----------
 autoconf_inc.m4                                  |  8 +++---
 build/aclocal/bakefile.m4                        |  9 +------
 build/bakefiles/config.bkl                       |  1 -
 build/bakefiles/wx.bkl                           | 23 ----------------
 build/msw/config.bcc                             |  2 +-
 build/msw/config.gcc                             |  2 +-
 build/msw/config.vc                              |  2 +-
 build/msw/makefile.bcc                           |  2 +-
 build/msw/makefile.gcc                           |  2 +-
 build/msw/makefile.vc                            |  2 +-
 build/msw/wx_vc7_adv.vcproj                      |  2 +-
 build/msw/wx_vc7_aui.vcproj                      |  2 +-
 build/msw/wx_vc7_base.vcproj                     |  2 +-
 build/msw/wx_vc7_core.vcproj                     |  2 +-
 build/msw/wx_vc7_gl.vcproj                       |  2 +-
 build/msw/wx_vc7_html.vcproj                     |  2 +-
 build/msw/wx_vc7_media.vcproj                    |  2 +-
 build/msw/wx_vc7_net.vcproj                      |  2 +-
 build/msw/wx_vc7_propgrid.vcproj                 |  2 +-
 build/msw/wx_vc7_qa.vcproj                       |  2 +-
 build/msw/wx_vc7_ribbon.vcproj                   |  2 +-
 build/msw/wx_vc7_richtext.vcproj                 |  2 +-
 build/msw/wx_vc7_stc.vcproj                      |  2 +-
 build/msw/wx_vc7_webview.vcproj                  |  2 +-
 build/msw/wx_vc7_wxexpat.vcproj                  |  2 +-
 build/msw/wx_vc7_wxjpeg.vcproj                   |  2 +-
 build/msw/wx_vc7_wxpng.vcproj                    |  2 +-
 build/msw/wx_vc7_wxregex.vcproj                  |  2 +-
 build/msw/wx_vc7_wxscintilla.vcproj              |  2 +-
 build/msw/wx_vc7_wxtiff.vcproj                   |  2 +-
 build/msw/wx_vc7_wxzlib.vcproj                   |  2 +-
 build/msw/wx_vc7_xml.vcproj                      |  2 +-
 build/msw/wx_vc7_xrc.vcproj                      |  2 +-
 build/msw/wx_vc8_adv.vcproj                      |  2 +-
 build/msw/wx_vc8_aui.vcproj                      |  2 +-
 build/msw/wx_vc8_base.vcproj                     |  2 +-
 build/msw/wx_vc8_core.vcproj                     |  2 +-
 build/msw/wx_vc8_gl.vcproj                       |  2 +-
 build/msw/wx_vc8_html.vcproj                     |  2 +-
 build/msw/wx_vc8_media.vcproj                    |  2 +-
 build/msw/wx_vc8_net.vcproj                      |  2 +-
 build/msw/wx_vc8_propgrid.vcproj                 |  2 +-
 build/msw/wx_vc8_qa.vcproj                       |  2 +-
 build/msw/wx_vc8_ribbon.vcproj                   |  2 +-
 build/msw/wx_vc8_richtext.vcproj                 |  2 +-
 build/msw/wx_vc8_stc.vcproj                      |  2 +-
 build/msw/wx_vc8_webview.vcproj                  |  2 +-
 build/msw/wx_vc8_wxexpat.vcproj                  |  2 +-
 build/msw/wx_vc8_wxjpeg.vcproj                   |  2 +-
 build/msw/wx_vc8_wxpng.vcproj                    |  2 +-
 build/msw/wx_vc8_wxregex.vcproj                  |  2 +-
 build/msw/wx_vc8_wxscintilla.vcproj              |  2 +-
 build/msw/wx_vc8_wxtiff.vcproj                   |  2 +-
 build/msw/wx_vc8_wxzlib.vcproj                   |  2 +-
 build/msw/wx_vc8_xml.vcproj                      |  2 +-
 build/msw/wx_vc8_xrc.vcproj                      |  2 +-
 build/msw/wx_vc9_adv.vcproj                      |  2 +-
 build/msw/wx_vc9_aui.vcproj                      |  2 +-
 build/msw/wx_vc9_base.vcproj                     |  2 +-
 build/msw/wx_vc9_core.vcproj                     |  2 +-
 build/msw/wx_vc9_gl.vcproj                       |  2 +-
 build/msw/wx_vc9_html.vcproj                     |  2 +-
 build/msw/wx_vc9_media.vcproj                    |  2 +-
 build/msw/wx_vc9_net.vcproj                      |  2 +-
 build/msw/wx_vc9_propgrid.vcproj                 |  2 +-
 build/msw/wx_vc9_qa.vcproj                       |  2 +-
 build/msw/wx_vc9_ribbon.vcproj                   |  2 +-
 build/msw/wx_vc9_richtext.vcproj                 |  2 +-
 build/msw/wx_vc9_stc.vcproj                      |  2 +-
 build/msw/wx_vc9_webview.vcproj                  |  2 +-
 build/msw/wx_vc9_wxexpat.vcproj                  |  2 +-
 build/msw/wx_vc9_wxjpeg.vcproj                   |  2 +-
 build/msw/wx_vc9_wxpng.vcproj                    |  2 +-
 build/msw/wx_vc9_wxregex.vcproj                  |  2 +-
 build/msw/wx_vc9_wxscintilla.vcproj              |  2 +-
 build/msw/wx_vc9_wxtiff.vcproj                   |  2 +-
 build/msw/wx_vc9_wxzlib.vcproj                   |  2 +-
 build/msw/wx_vc9_xml.vcproj                      |  2 +-
 build/msw/wx_vc9_xrc.vcproj                      |  2 +-
 demos/Makefile.in                                |  2 +-
 demos/bombs/Makefile.in                          | 24 +++++++----------
 demos/bombs/bombs_vc7.vcproj                     |  2 +-
 demos/bombs/bombs_vc8.vcproj                     |  2 +-
 demos/bombs/bombs_vc9.vcproj                     |  2 +-
 demos/bombs/makefile.bcc                         |  2 +-
 demos/bombs/makefile.gcc                         |  2 +-
 demos/bombs/makefile.vc                          |  2 +-
 demos/forty/Makefile.in                          | 24 +++++++----------
 demos/forty/forty_vc7.vcproj                     |  2 +-
 demos/forty/forty_vc8.vcproj                     |  2 +-
 demos/forty/forty_vc9.vcproj                     |  2 +-
 demos/forty/makefile.bcc                         |  2 +-
 demos/forty/makefile.gcc                         |  2 +-
 demos/forty/makefile.vc                          |  2 +-
 demos/fractal/Makefile.in                        | 24 +++++++----------
 demos/fractal/fractal_vc7.vcproj                 |  2 +-
 demos/fractal/fractal_vc8.vcproj                 |  2 +-
 demos/fractal/fractal_vc9.vcproj                 |  2 +-
 demos/fractal/makefile.bcc                       |  2 +-
 demos/fractal/makefile.gcc                       |  2 +-
 demos/fractal/makefile.vc                        |  2 +-
 demos/life/Makefile.in                           | 24 +++++++----------
 demos/life/life_vc7.vcproj                       |  2 +-
 demos/life/life_vc8.vcproj                       |  2 +-
 demos/life/life_vc9.vcproj                       |  2 +-
 demos/life/makefile.bcc                          |  2 +-
 demos/life/makefile.gcc                          |  2 +-
 demos/life/makefile.vc                           |  2 +-
 demos/makefile.bcc                               |  2 +-
 demos/makefile.gcc                               |  2 +-
 demos/makefile.vc                                |  2 +-
 demos/poem/Makefile.in                           | 24 +++++++----------
 demos/poem/makefile.bcc                          |  2 +-
 demos/poem/makefile.gcc                          |  2 +-
 demos/poem/makefile.vc                           |  2 +-
 demos/poem/poem_vc7.vcproj                       |  2 +-
 demos/poem/poem_vc8.vcproj                       |  2 +-
 demos/poem/poem_vc9.vcproj                       |  2 +-
 samples/access/Makefile.in                       | 24 +++++++----------
 samples/access/access_vc7.vcproj                 |  2 +-
 samples/access/access_vc8.vcproj                 |  2 +-
 samples/access/access_vc9.vcproj                 |  2 +-
 samples/access/makefile.bcc                      |  2 +-
 samples/access/makefile.gcc                      |  2 +-
 samples/access/makefile.unx                      |  2 +-
 samples/access/makefile.vc                       |  2 +-
 samples/animate/Makefile.in                      | 24 +++++++----------
 samples/animate/anitest_vc7.vcproj               |  2 +-
 samples/animate/anitest_vc8.vcproj               |  2 +-
 samples/animate/anitest_vc9.vcproj               |  2 +-
 samples/animate/makefile.bcc                     |  2 +-
 samples/animate/makefile.gcc                     |  2 +-
 samples/animate/makefile.unx                     |  2 +-
 samples/animate/makefile.vc                      |  2 +-
 samples/artprov/Makefile.in                      | 24 +++++++----------
 samples/artprov/artprov_vc7.vcproj               |  2 +-
 samples/artprov/artprov_vc8.vcproj               |  2 +-
 samples/artprov/artprov_vc9.vcproj               |  2 +-
 samples/artprov/makefile.bcc                     |  2 +-
 samples/artprov/makefile.gcc                     |  2 +-
 samples/artprov/makefile.unx                     |  2 +-
 samples/artprov/makefile.vc                      |  2 +-
 samples/aui/Makefile.in                          | 24 +++++++----------
 samples/aui/auidemo_vc7.vcproj                   |  2 +-
 samples/aui/auidemo_vc8.vcproj                   |  2 +-
 samples/aui/auidemo_vc9.vcproj                   |  2 +-
 samples/aui/makefile.bcc                         |  2 +-
 samples/aui/makefile.gcc                         |  2 +-
 samples/aui/makefile.unx                         |  2 +-
 samples/aui/makefile.vc                          |  2 +-
 samples/calendar/Makefile.in                     | 24 +++++++----------
 samples/calendar/calendar_vc7.vcproj             |  2 +-
 samples/calendar/calendar_vc8.vcproj             |  2 +-
 samples/calendar/calendar_vc9.vcproj             |  2 +-
 samples/calendar/makefile.bcc                    |  2 +-
 samples/calendar/makefile.gcc                    |  2 +-
 samples/calendar/makefile.unx                    |  2 +-
 samples/calendar/makefile.vc                     |  2 +-
 samples/caret/Makefile.in                        | 24 +++++++----------
 samples/caret/caret_vc7.vcproj                   |  2 +-
 samples/caret/caret_vc8.vcproj                   |  2 +-
 samples/caret/caret_vc9.vcproj                   |  2 +-
 samples/caret/makefile.bcc                       |  2 +-
 samples/caret/makefile.gcc                       |  2 +-
 samples/caret/makefile.unx                       |  2 +-
 samples/caret/makefile.vc                        |  2 +-
 samples/clipboard/Makefile.in                    | 24 +++++++----------
 samples/clipboard/clipboard_vc7.vcproj           |  2 +-
 samples/clipboard/clipboard_vc8.vcproj           |  2 +-
 samples/clipboard/clipboard_vc9.vcproj           |  2 +-
 samples/clipboard/makefile.bcc                   |  2 +-
 samples/clipboard/makefile.gcc                   |  2 +-
 samples/clipboard/makefile.unx                   |  2 +-
 samples/clipboard/makefile.vc                    |  2 +-
 samples/collpane/Makefile.in                     | 24 +++++++----------
 samples/collpane/collpane_vc7.vcproj             |  2 +-
 samples/collpane/collpane_vc8.vcproj             |  2 +-
 samples/collpane/collpane_vc9.vcproj             |  2 +-
 samples/collpane/makefile.bcc                    |  2 +-
 samples/collpane/makefile.gcc                    |  2 +-
 samples/collpane/makefile.unx                    |  2 +-
 samples/collpane/makefile.vc                     |  2 +-
 samples/combo/Makefile.in                        | 24 +++++++----------
 samples/combo/combo_vc7.vcproj                   |  2 +-
 samples/combo/combo_vc8.vcproj                   |  2 +-
 samples/combo/combo_vc9.vcproj                   |  2 +-
 samples/combo/makefile.bcc                       |  2 +-
 samples/combo/makefile.gcc                       |  2 +-
 samples/combo/makefile.unx                       |  2 +-
 samples/combo/makefile.vc                        |  2 +-
 samples/config/Makefile.in                       | 24 +++++++----------
 samples/config/config_vc7.vcproj                 |  2 +-
 samples/config/config_vc8.vcproj                 |  2 +-
 samples/config/config_vc9.vcproj                 |  2 +-
 samples/config/makefile.bcc                      |  2 +-
 samples/config/makefile.gcc                      |  2 +-
 samples/config/makefile.unx                      |  2 +-
 samples/config/makefile.vc                       |  2 +-
 samples/console/Makefile.in                      |  8 +-----
 samples/console/console_vc7.vcproj               |  2 +-
 samples/console/console_vc8.vcproj               |  2 +-
 samples/console/console_vc9.vcproj               |  2 +-
 samples/console/makefile.bcc                     |  2 +-
 samples/console/makefile.gcc                     |  2 +-
 samples/console/makefile.unx                     |  2 +-
 samples/console/makefile.vc                      |  2 +-
 samples/dataview/Makefile.in                     | 24 +++++++----------
 samples/dataview/dataview_vc7.vcproj             |  2 +-
 samples/dataview/dataview_vc8.vcproj             |  2 +-
 samples/dataview/dataview_vc9.vcproj             |  2 +-
 samples/dataview/makefile.bcc                    |  2 +-
 samples/dataview/makefile.gcc                    |  2 +-
 samples/dataview/makefile.unx                    |  2 +-
 samples/dataview/makefile.vc                     |  2 +-
 samples/debugrpt/Makefile.in                     | 24 +++++++----------
 samples/debugrpt/debugrpt_vc7.vcproj             |  2 +-
 samples/debugrpt/debugrpt_vc8.vcproj             |  2 +-
 samples/debugrpt/debugrpt_vc9.vcproj             |  2 +-
 samples/debugrpt/makefile.bcc                    |  2 +-
 samples/debugrpt/makefile.gcc                    |  2 +-
 samples/debugrpt/makefile.unx                    |  2 +-
 samples/debugrpt/makefile.vc                     |  2 +-
 samples/dialogs/Makefile.in                      | 24 +++++++----------
 samples/dialogs/dialogs_vc7.vcproj               |  2 +-
 samples/dialogs/dialogs_vc8.vcproj               |  2 +-
 samples/dialogs/dialogs_vc9.vcproj               |  2 +-
 samples/dialogs/makefile.bcc                     |  2 +-
 samples/dialogs/makefile.gcc                     |  2 +-
 samples/dialogs/makefile.unx                     |  2 +-
 samples/dialogs/makefile.vc                      |  2 +-
 samples/dialup/Makefile.in                       | 24 +++++++----------
 samples/dialup/dialup_vc7.vcproj                 |  2 +-
 samples/dialup/dialup_vc8.vcproj                 |  2 +-
 samples/dialup/dialup_vc9.vcproj                 |  2 +-
 samples/dialup/makefile.bcc                      |  2 +-
 samples/dialup/makefile.gcc                      |  2 +-
 samples/dialup/makefile.unx                      |  2 +-
 samples/dialup/makefile.vc                       |  2 +-
 samples/display/Makefile.in                      | 24 +++++++----------
 samples/display/display_vc7.vcproj               |  2 +-
 samples/display/display_vc8.vcproj               |  2 +-
 samples/display/display_vc9.vcproj               |  2 +-
 samples/display/makefile.bcc                     |  2 +-
 samples/display/makefile.gcc                     |  2 +-
 samples/display/makefile.unx                     |  2 +-
 samples/display/makefile.vc                      |  2 +-
 samples/dll/Makefile.in                          | 29 +++++++-------------
 samples/dll/dll_vc7_my_dll.vcproj                |  2 +-
 samples/dll/dll_vc7_sdk_exe.vcproj               |  2 +-
 samples/dll/dll_vc7_wx_exe.vcproj                |  2 +-
 samples/dll/dll_vc8_my_dll.vcproj                |  2 +-
 samples/dll/dll_vc8_sdk_exe.vcproj               |  2 +-
 samples/dll/dll_vc8_wx_exe.vcproj                |  2 +-
 samples/dll/dll_vc9_my_dll.vcproj                |  2 +-
 samples/dll/dll_vc9_sdk_exe.vcproj               |  2 +-
 samples/dll/dll_vc9_wx_exe.vcproj                |  2 +-
 samples/dll/makefile.bcc                         |  2 +-
 samples/dll/makefile.gcc                         |  2 +-
 samples/dll/makefile.unx                         |  2 +-
 samples/dll/makefile.vc                          |  2 +-
 samples/dnd/Makefile.in                          | 24 +++++++----------
 samples/dnd/dnd_vc7.vcproj                       |  2 +-
 samples/dnd/dnd_vc8.vcproj                       |  2 +-
 samples/dnd/dnd_vc9.vcproj                       |  2 +-
 samples/dnd/makefile.bcc                         |  2 +-
 samples/dnd/makefile.gcc                         |  2 +-
 samples/dnd/makefile.unx                         |  2 +-
 samples/dnd/makefile.vc                          |  2 +-
 samples/docview/Makefile.in                      | 24 +++++++----------
 samples/docview/docview_vc7.vcproj               |  2 +-
 samples/docview/docview_vc8.vcproj               |  2 +-
 samples/docview/docview_vc9.vcproj               |  2 +-
 samples/docview/makefile.bcc                     |  2 +-
 samples/docview/makefile.gcc                     |  2 +-
 samples/docview/makefile.unx                     |  2 +-
 samples/docview/makefile.vc                      |  2 +-
 samples/dragimag/Makefile.in                     | 24 +++++++----------
 samples/dragimag/dragimag_vc7.vcproj             |  2 +-
 samples/dragimag/dragimag_vc8.vcproj             |  2 +-
 samples/dragimag/dragimag_vc9.vcproj             |  2 +-
 samples/dragimag/makefile.bcc                    |  2 +-
 samples/dragimag/makefile.gcc                    |  2 +-
 samples/dragimag/makefile.unx                    |  2 +-
 samples/dragimag/makefile.vc                     |  2 +-
 samples/drawing/Makefile.in                      | 24 +++++++----------
 samples/drawing/drawing_vc7.vcproj               |  2 +-
 samples/drawing/drawing_vc8.vcproj               |  2 +-
 samples/drawing/drawing_vc9.vcproj               |  2 +-
 samples/drawing/makefile.bcc                     |  2 +-
 samples/drawing/makefile.gcc                     |  2 +-
 samples/drawing/makefile.unx                     |  2 +-
 samples/drawing/makefile.vc                      |  2 +-
 samples/erase/Makefile.in                        | 24 +++++++----------
 samples/erase/erase_vc7.vcproj                   |  2 +-
 samples/erase/erase_vc8.vcproj                   |  2 +-
 samples/erase/erase_vc9.vcproj                   |  2 +-
 samples/erase/makefile.bcc                       |  2 +-
 samples/erase/makefile.gcc                       |  2 +-
 samples/erase/makefile.unx                       |  2 +-
 samples/erase/makefile.vc                        |  2 +-
 samples/event/Makefile.in                        | 24 +++++++----------
 samples/event/event_vc7.vcproj                   |  2 +-
 samples/event/event_vc8.vcproj                   |  2 +-
 samples/event/event_vc9.vcproj                   |  2 +-
 samples/event/makefile.bcc                       |  2 +-
 samples/event/makefile.gcc                       |  2 +-
 samples/event/makefile.unx                       |  2 +-
 samples/event/makefile.vc                        |  2 +-
 samples/except/Makefile.in                       | 24 +++++++----------
 samples/except/except_vc7.vcproj                 |  2 +-
 samples/except/except_vc8.vcproj                 |  2 +-
 samples/except/except_vc9.vcproj                 |  2 +-
 samples/except/makefile.bcc                      |  2 +-
 samples/except/makefile.gcc                      |  2 +-
 samples/except/makefile.unx                      |  2 +-
 samples/except/makefile.vc                       |  2 +-
 samples/exec/Makefile.in                         | 24 +++++++----------
 samples/exec/exec_vc7.vcproj                     |  2 +-
 samples/exec/exec_vc8.vcproj                     |  2 +-
 samples/exec/exec_vc9.vcproj                     |  2 +-
 samples/exec/makefile.bcc                        |  2 +-
 samples/exec/makefile.gcc                        |  2 +-
 samples/exec/makefile.unx                        |  2 +-
 samples/exec/makefile.vc                         |  2 +-
 samples/flash/flash_vc7.vcproj                   |  2 +-
 samples/flash/flash_vc8.vcproj                   |  2 +-
 samples/flash/flash_vc9.vcproj                   |  2 +-
 samples/flash/makefile.vc                        |  2 +-
 samples/font/Makefile.in                         | 24 +++++++----------
 samples/font/font_vc7.vcproj                     |  2 +-
 samples/font/font_vc8.vcproj                     |  2 +-
 samples/font/font_vc9.vcproj                     |  2 +-
 samples/font/makefile.bcc                        |  2 +-
 samples/font/makefile.gcc                        |  2 +-
 samples/font/makefile.unx                        |  2 +-
 samples/font/makefile.vc                         |  2 +-
 samples/fswatcher/Makefile.in                    | 24 +++++++----------
 samples/fswatcher/fswatcher_vc7.vcproj           |  2 +-
 samples/fswatcher/fswatcher_vc8.vcproj           |  2 +-
 samples/fswatcher/fswatcher_vc9.vcproj           |  2 +-
 samples/fswatcher/makefile.bcc                   |  2 +-
 samples/fswatcher/makefile.gcc                   |  2 +-
 samples/fswatcher/makefile.unx                   |  2 +-
 samples/fswatcher/makefile.vc                    |  2 +-
 samples/grid/Makefile.in                         | 24 +++++++----------
 samples/grid/grid_vc7.vcproj                     |  2 +-
 samples/grid/grid_vc8.vcproj                     |  2 +-
 samples/grid/grid_vc9.vcproj                     |  2 +-
 samples/grid/makefile.bcc                        |  2 +-
 samples/grid/makefile.gcc                        |  2 +-
 samples/grid/makefile.unx                        |  2 +-
 samples/grid/makefile.vc                         |  2 +-
 samples/help/Makefile.in                         | 24 +++++++----------
 samples/help/help_vc7.vcproj                     |  2 +-
 samples/help/help_vc8.vcproj                     |  2 +-
 samples/help/help_vc9.vcproj                     |  2 +-
 samples/help/makefile.bcc                        |  2 +-
 samples/help/makefile.gcc                        |  2 +-
 samples/help/makefile.unx                        |  2 +-
 samples/help/makefile.vc                         |  2 +-
 samples/htlbox/Makefile.in                       | 24 +++++++----------
 samples/htlbox/htlbox_vc7.vcproj                 |  2 +-
 samples/htlbox/htlbox_vc8.vcproj                 |  2 +-
 samples/htlbox/htlbox_vc9.vcproj                 |  2 +-
 samples/htlbox/makefile.bcc                      |  2 +-
 samples/htlbox/makefile.gcc                      |  2 +-
 samples/htlbox/makefile.unx                      |  2 +-
 samples/htlbox/makefile.vc                       |  2 +-
 samples/html/Makefile.in                         |  2 +-
 samples/html/about/Makefile.in                   | 24 +++++++----------
 samples/html/about/about_vc7.vcproj              |  2 +-
 samples/html/about/about_vc8.vcproj              |  2 +-
 samples/html/about/about_vc9.vcproj              |  2 +-
 samples/html/about/makefile.bcc                  |  2 +-
 samples/html/about/makefile.gcc                  |  2 +-
 samples/html/about/makefile.unx                  |  2 +-
 samples/html/about/makefile.vc                   |  2 +-
 samples/html/help/Makefile.in                    | 24 +++++++----------
 samples/html/help/help_vc7.vcproj                |  2 +-
 samples/html/help/help_vc8.vcproj                |  2 +-
 samples/html/help/help_vc9.vcproj                |  2 +-
 samples/html/help/makefile.bcc                   |  2 +-
 samples/html/help/makefile.gcc                   |  2 +-
 samples/html/help/makefile.unx                   |  2 +-
 samples/html/help/makefile.vc                    |  2 +-
 samples/html/helpview/Makefile.in                | 24 +++++++----------
 samples/html/helpview/helpview_vc7.vcproj        |  2 +-
 samples/html/helpview/helpview_vc8.vcproj        |  2 +-
 samples/html/helpview/helpview_vc9.vcproj        |  2 +-
 samples/html/helpview/makefile.bcc               |  2 +-
 samples/html/helpview/makefile.gcc               |  2 +-
 samples/html/helpview/makefile.unx               |  2 +-
 samples/html/helpview/makefile.vc                |  2 +-
 samples/html/htmlctrl/Makefile.in                | 24 +++++++----------
 samples/html/htmlctrl/makefile.unx               |  2 +-
 samples/html/makefile.bcc                        |  2 +-
 samples/html/makefile.gcc                        |  2 +-
 samples/html/makefile.unx                        |  2 +-
 samples/html/makefile.vc                         |  2 +-
 samples/html/printing/Makefile.in                | 24 +++++++----------
 samples/html/printing/makefile.bcc               |  2 +-
 samples/html/printing/makefile.gcc               |  2 +-
 samples/html/printing/makefile.unx               |  2 +-
 samples/html/printing/makefile.vc                |  2 +-
 samples/html/printing/printing_vc7.vcproj        |  2 +-
 samples/html/printing/printing_vc8.vcproj        |  2 +-
 samples/html/printing/printing_vc9.vcproj        |  2 +-
 samples/html/test/Makefile.in                    | 24 +++++++----------
 samples/html/test/makefile.bcc                   |  2 +-
 samples/html/test/makefile.gcc                   |  2 +-
 samples/html/test/makefile.unx                   |  2 +-
 samples/html/test/makefile.vc                    |  2 +-
 samples/html/test/test_vc7.vcproj                |  2 +-
 samples/html/test/test_vc8.vcproj                |  2 +-
 samples/html/test/test_vc9.vcproj                |  2 +-
 samples/html/virtual/Makefile.in                 | 24 +++++++----------
 samples/html/virtual/makefile.bcc                |  2 +-
 samples/html/virtual/makefile.gcc                |  2 +-
 samples/html/virtual/makefile.unx                |  2 +-
 samples/html/virtual/makefile.vc                 |  2 +-
 samples/html/virtual/virtual_vc7.vcproj          |  2 +-
 samples/html/virtual/virtual_vc8.vcproj          |  2 +-
 samples/html/virtual/virtual_vc9.vcproj          |  2 +-
 samples/html/widget/Makefile.in                  | 24 +++++++----------
 samples/html/widget/makefile.bcc                 |  2 +-
 samples/html/widget/makefile.gcc                 |  2 +-
 samples/html/widget/makefile.unx                 |  2 +-
 samples/html/widget/makefile.vc                  |  2 +-
 samples/html/widget/widget_vc7.vcproj            |  2 +-
 samples/html/widget/widget_vc8.vcproj            |  2 +-
 samples/html/widget/widget_vc9.vcproj            |  2 +-
 samples/html/zip/Makefile.in                     | 24 +++++++----------
 samples/html/zip/makefile.bcc                    |  2 +-
 samples/html/zip/makefile.gcc                    |  2 +-
 samples/html/zip/makefile.unx                    |  2 +-
 samples/html/zip/makefile.vc                     |  2 +-
 samples/html/zip/zip_vc7.vcproj                  |  2 +-
 samples/html/zip/zip_vc8.vcproj                  |  2 +-
 samples/html/zip/zip_vc9.vcproj                  |  2 +-
 samples/image/Makefile.in                        | 24 +++++++----------
 samples/image/image_vc7.vcproj                   |  2 +-
 samples/image/image_vc8.vcproj                   |  2 +-
 samples/image/image_vc9.vcproj                   |  2 +-
 samples/image/makefile.bcc                       |  2 +-
 samples/image/makefile.gcc                       |  2 +-
 samples/image/makefile.unx                       |  2 +-
 samples/image/makefile.vc                        |  2 +-
 samples/internat/Makefile.in                     | 24 +++++++----------
 samples/internat/internat_vc7.vcproj             |  2 +-
 samples/internat/internat_vc8.vcproj             |  2 +-
 samples/internat/internat_vc9.vcproj             |  2 +-
 samples/internat/makefile.bcc                    |  2 +-
 samples/internat/makefile.gcc                    |  2 +-
 samples/internat/makefile.unx                    |  2 +-
 samples/internat/makefile.vc                     |  2 +-
 samples/ipc/Makefile.in                          | 23 +---------------
 samples/ipc/ipc_vc7_baseipcclient.vcproj         |  2 +-
 samples/ipc/ipc_vc7_baseipcserver.vcproj         |  2 +-
 samples/ipc/ipc_vc7_ipcclient.vcproj             |  2 +-
 samples/ipc/ipc_vc7_ipcserver.vcproj             |  2 +-
 samples/ipc/ipc_vc8_baseipcclient.vcproj         |  2 +-
 samples/ipc/ipc_vc8_baseipcserver.vcproj         |  2 +-
 samples/ipc/ipc_vc8_ipcclient.vcproj             |  2 +-
 samples/ipc/ipc_vc8_ipcserver.vcproj             |  2 +-
 samples/ipc/ipc_vc9_baseipcclient.vcproj         |  2 +-
 samples/ipc/ipc_vc9_baseipcserver.vcproj         |  2 +-
 samples/ipc/ipc_vc9_ipcclient.vcproj             |  2 +-
 samples/ipc/ipc_vc9_ipcserver.vcproj             |  2 +-
 samples/ipc/makefile.bcc                         |  2 +-
 samples/ipc/makefile.gcc                         |  2 +-
 samples/ipc/makefile.unx                         |  2 +-
 samples/ipc/makefile.vc                          |  2 +-
 samples/joytest/Makefile.in                      | 24 +++++++----------
 samples/joytest/joytest_vc7.vcproj               |  2 +-
 samples/joytest/joytest_vc8.vcproj               |  2 +-
 samples/joytest/joytest_vc9.vcproj               |  2 +-
 samples/joytest/makefile.bcc                     |  2 +-
 samples/joytest/makefile.gcc                     |  2 +-
 samples/joytest/makefile.unx                     |  2 +-
 samples/joytest/makefile.vc                      |  2 +-
 samples/keyboard/Makefile.in                     | 24 +++++++----------
 samples/keyboard/keyboard_vc7.vcproj             |  2 +-
 samples/keyboard/keyboard_vc8.vcproj             |  2 +-
 samples/keyboard/keyboard_vc9.vcproj             |  2 +-
 samples/keyboard/makefile.bcc                    |  2 +-
 samples/keyboard/makefile.gcc                    |  2 +-
 samples/keyboard/makefile.unx                    |  2 +-
 samples/keyboard/makefile.vc                     |  2 +-
 samples/layout/Makefile.in                       | 24 +++++++----------
 samples/layout/layout_vc7.vcproj                 |  2 +-
 samples/layout/layout_vc8.vcproj                 |  2 +-
 samples/layout/layout_vc9.vcproj                 |  2 +-
 samples/layout/makefile.bcc                      |  2 +-
 samples/layout/makefile.gcc                      |  2 +-
 samples/layout/makefile.unx                      |  2 +-
 samples/layout/makefile.vc                       |  2 +-
 samples/listctrl/Makefile.in                     | 24 +++++++----------
 samples/listctrl/listctrl_vc7.vcproj             |  2 +-
 samples/listctrl/listctrl_vc8.vcproj             |  2 +-
 samples/listctrl/listctrl_vc9.vcproj             |  2 +-
 samples/listctrl/makefile.bcc                    |  2 +-
 samples/listctrl/makefile.gcc                    |  2 +-
 samples/listctrl/makefile.unx                    |  2 +-
 samples/listctrl/makefile.vc                     |  2 +-
 samples/makefile.bcc                             |  2 +-
 samples/makefile.gcc                             |  2 +-
 samples/makefile.vc                              |  2 +-
 samples/mdi/Makefile.in                          | 24 +++++++----------
 samples/mdi/makefile.bcc                         |  2 +-
 samples/mdi/makefile.gcc                         |  2 +-
 samples/mdi/makefile.unx                         |  2 +-
 samples/mdi/makefile.vc                          |  2 +-
 samples/mdi/mdi_vc7.vcproj                       |  2 +-
 samples/mdi/mdi_vc8.vcproj                       |  2 +-
 samples/mdi/mdi_vc9.vcproj                       |  2 +-
 samples/mediaplayer/Makefile.in                  | 24 +++++++----------
 samples/mediaplayer/makefile.bcc                 |  2 +-
 samples/mediaplayer/makefile.gcc                 |  2 +-
 samples/mediaplayer/makefile.unx                 |  2 +-
 samples/mediaplayer/makefile.vc                  |  2 +-
 samples/mediaplayer/mediaplayer_vc7.vcproj       |  2 +-
 samples/mediaplayer/mediaplayer_vc8.vcproj       |  2 +-
 samples/mediaplayer/mediaplayer_vc9.vcproj       |  2 +-
 samples/memcheck/Makefile.in                     | 24 +++++++----------
 samples/memcheck/makefile.bcc                    |  2 +-
 samples/memcheck/makefile.gcc                    |  2 +-
 samples/memcheck/makefile.unx                    |  2 +-
 samples/memcheck/makefile.vc                     |  2 +-
 samples/memcheck/memcheck_vc7.vcproj             |  2 +-
 samples/memcheck/memcheck_vc8.vcproj             |  2 +-
 samples/memcheck/memcheck_vc9.vcproj             |  2 +-
 samples/menu/Makefile.in                         | 24 +++++++----------
 samples/menu/makefile.bcc                        |  2 +-
 samples/menu/makefile.gcc                        |  2 +-
 samples/menu/makefile.unx                        |  2 +-
 samples/menu/makefile.vc                         |  2 +-
 samples/menu/menu_vc7.vcproj                     |  2 +-
 samples/menu/menu_vc8.vcproj                     |  2 +-
 samples/menu/menu_vc9.vcproj                     |  2 +-
 samples/mfc/makefile.vc                          |  2 +-
 samples/mfc/mfc_vc7.vcproj                       |  2 +-
 samples/mfc/mfc_vc8.vcproj                       |  2 +-
 samples/mfc/mfc_vc9.vcproj                       |  2 +-
 samples/minimal/Makefile.in                      | 24 +++++++----------
 samples/minimal/makefile.bcc                     |  2 +-
 samples/minimal/makefile.gcc                     |  2 +-
 samples/minimal/makefile.unx                     |  2 +-
 samples/minimal/makefile.vc                      |  2 +-
 samples/minimal/minimal_vc7.vcproj               |  2 +-
 samples/minimal/minimal_vc8.vcproj               |  2 +-
 samples/minimal/minimal_vc9.vcproj               |  2 +-
 samples/nativdlg/Makefile.in                     | 24 +++++++----------
 samples/nativdlg/makefile.bcc                    |  2 +-
 samples/nativdlg/makefile.gcc                    |  2 +-
 samples/nativdlg/makefile.unx                    |  2 +-
 samples/nativdlg/makefile.vc                     |  2 +-
 samples/nativdlg/nativdlg_vc7.vcproj             |  2 +-
 samples/nativdlg/nativdlg_vc8.vcproj             |  2 +-
 samples/nativdlg/nativdlg_vc9.vcproj             |  2 +-
 samples/notebook/Makefile.in                     | 24 +++++++----------
 samples/notebook/makefile.bcc                    |  2 +-
 samples/notebook/makefile.gcc                    |  2 +-
 samples/notebook/makefile.unx                    |  2 +-
 samples/notebook/makefile.vc                     |  2 +-
 samples/notebook/notebook_vc7.vcproj             |  2 +-
 samples/notebook/notebook_vc8.vcproj             |  2 +-
 samples/notebook/notebook_vc9.vcproj             |  2 +-
 samples/oleauto/Makefile.in                      | 24 +++++++----------
 samples/oleauto/makefile.bcc                     |  2 +-
 samples/oleauto/makefile.gcc                     |  2 +-
 samples/oleauto/makefile.unx                     |  2 +-
 samples/oleauto/makefile.vc                      |  2 +-
 samples/oleauto/oleauto_vc7.vcproj               |  2 +-
 samples/oleauto/oleauto_vc8.vcproj               |  2 +-
 samples/oleauto/oleauto_vc9.vcproj               |  2 +-
 samples/opengl/Makefile.in                       |  2 +-
 samples/opengl/cube/Makefile.in                  | 24 +++++++----------
 samples/opengl/cube/cube_vc7.vcproj              |  2 +-
 samples/opengl/cube/cube_vc8.vcproj              |  2 +-
 samples/opengl/cube/cube_vc9.vcproj              |  2 +-
 samples/opengl/cube/makefile.bcc                 |  2 +-
 samples/opengl/cube/makefile.gcc                 |  2 +-
 samples/opengl/cube/makefile.unx                 |  2 +-
 samples/opengl/cube/makefile.vc                  |  2 +-
 samples/opengl/isosurf/Makefile.in               | 24 +++++++----------
 samples/opengl/isosurf/isosurf_vc7.vcproj        |  2 +-
 samples/opengl/isosurf/isosurf_vc8.vcproj        |  2 +-
 samples/opengl/isosurf/isosurf_vc9.vcproj        |  2 +-
 samples/opengl/isosurf/makefile.bcc              |  2 +-
 samples/opengl/isosurf/makefile.gcc              |  2 +-
 samples/opengl/isosurf/makefile.unx              |  2 +-
 samples/opengl/isosurf/makefile.vc               |  2 +-
 samples/opengl/makefile.bcc                      |  2 +-
 samples/opengl/makefile.gcc                      |  2 +-
 samples/opengl/makefile.unx                      |  2 +-
 samples/opengl/makefile.vc                       |  2 +-
 samples/opengl/penguin/Makefile.in               | 24 +++++++----------
 samples/opengl/penguin/makefile.bcc              |  2 +-
 samples/opengl/penguin/makefile.gcc              |  2 +-
 samples/opengl/penguin/makefile.unx              |  2 +-
 samples/opengl/penguin/makefile.vc               |  2 +-
 samples/opengl/penguin/penguin_vc7.vcproj        |  2 +-
 samples/opengl/penguin/penguin_vc8.vcproj        |  2 +-
 samples/opengl/penguin/penguin_vc9.vcproj        |  2 +-
 samples/opengl/pyramid/Makefile.in               | 24 +++++++----------
 samples/opengl/pyramid/makefile.bcc              |  2 +-
 samples/opengl/pyramid/makefile.gcc              |  2 +-
 samples/opengl/pyramid/makefile.unx              |  2 +-
 samples/opengl/pyramid/makefile.vc               |  2 +-
 samples/opengl/pyramid/pyramid_vc7.vcproj        |  2 +-
 samples/opengl/pyramid/pyramid_vc8.vcproj        |  2 +-
 samples/opengl/pyramid/pyramid_vc9.vcproj        |  2 +-
 samples/ownerdrw/Makefile.in                     | 24 +++++++----------
 samples/ownerdrw/makefile.bcc                    |  2 +-
 samples/ownerdrw/makefile.gcc                    |  2 +-
 samples/ownerdrw/makefile.unx                    |  2 +-
 samples/ownerdrw/makefile.vc                     |  2 +-
 samples/ownerdrw/ownerdrw_vc7.vcproj             |  2 +-
 samples/ownerdrw/ownerdrw_vc8.vcproj             |  2 +-
 samples/ownerdrw/ownerdrw_vc9.vcproj             |  2 +-
 samples/popup/Makefile.in                        | 24 +++++++----------
 samples/popup/makefile.bcc                       |  2 +-
 samples/popup/makefile.gcc                       |  2 +-
 samples/popup/makefile.unx                       |  2 +-
 samples/popup/makefile.vc                        |  2 +-
 samples/popup/popup_vc7.vcproj                   |  2 +-
 samples/popup/popup_vc8.vcproj                   |  2 +-
 samples/popup/popup_vc9.vcproj                   |  2 +-
 samples/power/Makefile.in                        | 24 +++++++----------
 samples/power/makefile.bcc                       |  2 +-
 samples/power/makefile.gcc                       |  2 +-
 samples/power/makefile.unx                       |  2 +-
 samples/power/makefile.vc                        |  2 +-
 samples/power/power_vc7.vcproj                   |  2 +-
 samples/power/power_vc8.vcproj                   |  2 +-
 samples/power/power_vc9.vcproj                   |  2 +-
 samples/preferences/Makefile.in                  | 24 +++++++----------
 samples/preferences/makefile.bcc                 |  2 +-
 samples/preferences/makefile.gcc                 |  2 +-
 samples/preferences/makefile.unx                 |  2 +-
 samples/preferences/makefile.vc                  |  2 +-
 samples/preferences/preferences_vc7.vcproj       |  2 +-
 samples/preferences/preferences_vc8.vcproj       |  2 +-
 samples/preferences/preferences_vc9.vcproj       |  2 +-
 samples/printing/Makefile.in                     | 24 +++++++----------
 samples/printing/makefile.bcc                    |  2 +-
 samples/printing/makefile.gcc                    |  2 +-
 samples/printing/makefile.unx                    |  2 +-
 samples/printing/makefile.vc                     |  2 +-
 samples/printing/printing_vc7.vcproj             |  2 +-
 samples/printing/printing_vc8.vcproj             |  2 +-
 samples/printing/printing_vc9.vcproj             |  2 +-
 samples/propgrid/Makefile.in                     | 24 +++++++----------
 samples/propgrid/makefile.bcc                    |  2 +-
 samples/propgrid/makefile.gcc                    |  2 +-
 samples/propgrid/makefile.unx                    |  2 +-
 samples/propgrid/makefile.vc                     |  2 +-
 samples/propgrid/propgrid_vc7.vcproj             |  2 +-
 samples/propgrid/propgrid_vc8.vcproj             |  2 +-
 samples/propgrid/propgrid_vc9.vcproj             |  2 +-
 samples/regtest/Makefile.in                      | 24 +++++++----------
 samples/regtest/makefile.bcc                     |  2 +-
 samples/regtest/makefile.gcc                     |  2 +-
 samples/regtest/makefile.unx                     |  2 +-
 samples/regtest/makefile.vc                      |  2 +-
 samples/regtest/regtest_vc7.vcproj               |  2 +-
 samples/regtest/regtest_vc8.vcproj               |  2 +-
 samples/regtest/regtest_vc9.vcproj               |  2 +-
 samples/render/Makefile.in                       | 24 +++++++----------
 samples/render/makefile.bcc                      |  2 +-
 samples/render/makefile.gcc                      |  2 +-
 samples/render/makefile.unx                      |  2 +-
 samples/render/makefile.vc                       |  2 +-
 samples/render/render_vc7_renddll.vcproj         |  2 +-
 samples/render/render_vc7_render.vcproj          |  2 +-
 samples/render/render_vc8_renddll.vcproj         |  2 +-
 samples/render/render_vc8_render.vcproj          |  2 +-
 samples/render/render_vc9_renddll.vcproj         |  2 +-
 samples/render/render_vc9_render.vcproj          |  2 +-
 samples/ribbon/Makefile.in                       | 24 +++++++----------
 samples/ribbon/makefile.bcc                      |  2 +-
 samples/ribbon/makefile.gcc                      |  2 +-
 samples/ribbon/makefile.unx                      |  2 +-
 samples/ribbon/makefile.vc                       |  2 +-
 samples/ribbon/ribbon_vc7.vcproj                 |  2 +-
 samples/ribbon/ribbon_vc8.vcproj                 |  2 +-
 samples/ribbon/ribbon_vc9.vcproj                 |  2 +-
 samples/richtext/Makefile.in                     | 24 +++++++----------
 samples/richtext/makefile.bcc                    |  2 +-
 samples/richtext/makefile.gcc                    |  2 +-
 samples/richtext/makefile.unx                    |  2 +-
 samples/richtext/makefile.vc                     |  2 +-
 samples/richtext/richtext_vc7.vcproj             |  2 +-
 samples/richtext/richtext_vc8.vcproj             |  2 +-
 samples/richtext/richtext_vc9.vcproj             |  2 +-
 samples/sashtest/Makefile.in                     | 24 +++++++----------
 samples/sashtest/makefile.bcc                    |  2 +-
 samples/sashtest/makefile.gcc                    |  2 +-
 samples/sashtest/makefile.unx                    |  2 +-
 samples/sashtest/makefile.vc                     |  2 +-
 samples/sashtest/sashtest_vc7.vcproj             |  2 +-
 samples/sashtest/sashtest_vc8.vcproj             |  2 +-
 samples/sashtest/sashtest_vc9.vcproj             |  2 +-
 samples/scroll/Makefile.in                       | 24 +++++++----------
 samples/scroll/makefile.bcc                      |  2 +-
 samples/scroll/makefile.gcc                      |  2 +-
 samples/scroll/makefile.unx                      |  2 +-
 samples/scroll/makefile.vc                       |  2 +-
 samples/scroll/scroll_vc7.vcproj                 |  2 +-
 samples/scroll/scroll_vc8.vcproj                 |  2 +-
 samples/scroll/scroll_vc9.vcproj                 |  2 +-
 samples/secretstore/Makefile.in                  |  8 +-----
 samples/secretstore/makefile.bcc                 |  2 +-
 samples/secretstore/makefile.gcc                 |  2 +-
 samples/secretstore/makefile.unx                 |  2 +-
 samples/secretstore/makefile.vc                  |  2 +-
 samples/secretstore/secretstore_vc7.vcproj       |  2 +-
 samples/secretstore/secretstore_vc8.vcproj       |  2 +-
 samples/secretstore/secretstore_vc9.vcproj       |  2 +-
 samples/shaped/Makefile.in                       | 24 +++++++----------
 samples/shaped/makefile.bcc                      |  2 +-
 samples/shaped/makefile.gcc                      |  2 +-
 samples/shaped/makefile.unx                      |  2 +-
 samples/shaped/makefile.vc                       |  2 +-
 samples/shaped/shaped_vc7.vcproj                 |  2 +-
 samples/shaped/shaped_vc8.vcproj                 |  2 +-
 samples/shaped/shaped_vc9.vcproj                 |  2 +-
 samples/sockets/Makefile.in                      | 23 +---------------
 samples/sockets/makefile.bcc                     |  2 +-
 samples/sockets/makefile.gcc                     |  2 +-
 samples/sockets/makefile.unx                     |  2 +-
 samples/sockets/makefile.vc                      |  2 +-
 samples/sockets/sockets_vc7_baseclient.vcproj    |  2 +-
 samples/sockets/sockets_vc7_baseserver.vcproj    |  2 +-
 samples/sockets/sockets_vc7_client.vcproj        |  2 +-
 samples/sockets/sockets_vc7_server.vcproj        |  2 +-
 samples/sockets/sockets_vc8_baseclient.vcproj    |  2 +-
 samples/sockets/sockets_vc8_baseserver.vcproj    |  2 +-
 samples/sockets/sockets_vc8_client.vcproj        |  2 +-
 samples/sockets/sockets_vc8_server.vcproj        |  2 +-
 samples/sockets/sockets_vc9_baseclient.vcproj    |  2 +-
 samples/sockets/sockets_vc9_baseserver.vcproj    |  2 +-
 samples/sockets/sockets_vc9_client.vcproj        |  2 +-
 samples/sockets/sockets_vc9_server.vcproj        |  2 +-
 samples/sound/Makefile.in                        | 24 +++++++----------
 samples/sound/makefile.bcc                       |  2 +-
 samples/sound/makefile.gcc                       |  2 +-
 samples/sound/makefile.unx                       |  2 +-
 samples/sound/makefile.vc                        |  2 +-
 samples/sound/sound_vc7.vcproj                   |  2 +-
 samples/sound/sound_vc8.vcproj                   |  2 +-
 samples/sound/sound_vc9.vcproj                   |  2 +-
 samples/splash/Makefile.in                       | 24 +++++++----------
 samples/splash/makefile.bcc                      |  2 +-
 samples/splash/makefile.gcc                      |  2 +-
 samples/splash/makefile.unx                      |  2 +-
 samples/splash/makefile.vc                       |  2 +-
 samples/splash/splash_vc7.vcproj                 |  2 +-
 samples/splash/splash_vc8.vcproj                 |  2 +-
 samples/splash/splash_vc9.vcproj                 |  2 +-
 samples/splitter/Makefile.in                     | 24 +++++++----------
 samples/splitter/makefile.bcc                    |  2 +-
 samples/splitter/makefile.gcc                    |  2 +-
 samples/splitter/makefile.unx                    |  2 +-
 samples/splitter/makefile.vc                     |  2 +-
 samples/splitter/splitter_vc7.vcproj             |  2 +-
 samples/splitter/splitter_vc8.vcproj             |  2 +-
 samples/splitter/splitter_vc9.vcproj             |  2 +-
 samples/statbar/Makefile.in                      | 24 +++++++----------
 samples/statbar/makefile.bcc                     |  2 +-
 samples/statbar/makefile.gcc                     |  2 +-
 samples/statbar/makefile.unx                     |  2 +-
 samples/statbar/makefile.vc                      |  2 +-
 samples/statbar/statbar_vc7.vcproj               |  2 +-
 samples/statbar/statbar_vc8.vcproj               |  2 +-
 samples/statbar/statbar_vc9.vcproj               |  2 +-
 samples/stc/Makefile.in                          | 24 +++++++----------
 samples/stc/makefile.bcc                         |  2 +-
 samples/stc/makefile.gcc                         |  2 +-
 samples/stc/makefile.unx                         |  2 +-
 samples/stc/makefile.vc                          |  2 +-
 samples/stc/stctest_vc7.vcproj                   |  2 +-
 samples/stc/stctest_vc8.vcproj                   |  2 +-
 samples/stc/stctest_vc9.vcproj                   |  2 +-
 samples/svg/Makefile.in                          | 24 +++++++----------
 samples/svg/makefile.bcc                         |  2 +-
 samples/svg/makefile.gcc                         |  2 +-
 samples/svg/makefile.unx                         |  2 +-
 samples/svg/makefile.vc                          |  2 +-
 samples/svg/svgtest_vc7.vcproj                   |  2 +-
 samples/svg/svgtest_vc8.vcproj                   |  2 +-
 samples/svg/svgtest_vc9.vcproj                   |  2 +-
 samples/taborder/Makefile.in                     | 24 +++++++----------
 samples/taborder/makefile.bcc                    |  2 +-
 samples/taborder/makefile.gcc                    |  2 +-
 samples/taborder/makefile.unx                    |  2 +-
 samples/taborder/makefile.vc                     |  2 +-
 samples/taborder/taborder_vc7.vcproj             |  2 +-
 samples/taborder/taborder_vc8.vcproj             |  2 +-
 samples/taborder/taborder_vc9.vcproj             |  2 +-
 samples/taskbar/Makefile.in                      | 24 +++++++----------
 samples/taskbar/makefile.bcc                     |  2 +-
 samples/taskbar/makefile.gcc                     |  2 +-
 samples/taskbar/makefile.unx                     |  2 +-
 samples/taskbar/makefile.vc                      |  2 +-
 samples/taskbar/taskbar_vc7.vcproj               |  2 +-
 samples/taskbar/taskbar_vc8.vcproj               |  2 +-
 samples/taskbar/taskbar_vc9.vcproj               |  2 +-
 samples/taskbarbutton/Makefile.in                | 24 +++++++----------
 samples/taskbarbutton/makefile.bcc               |  2 +-
 samples/taskbarbutton/makefile.gcc               |  2 +-
 samples/taskbarbutton/makefile.unx               |  2 +-
 samples/taskbarbutton/makefile.vc                |  2 +-
 samples/taskbarbutton/taskbarbutton_vc7.vcproj   |  2 +-
 samples/taskbarbutton/taskbarbutton_vc8.vcproj   |  2 +-
 samples/taskbarbutton/taskbarbutton_vc9.vcproj   |  2 +-
 samples/text/Makefile.in                         | 24 +++++++----------
 samples/text/makefile.bcc                        |  2 +-
 samples/text/makefile.gcc                        |  2 +-
 samples/text/makefile.unx                        |  2 +-
 samples/text/makefile.vc                         |  2 +-
 samples/text/text_vc7.vcproj                     |  2 +-
 samples/text/text_vc8.vcproj                     |  2 +-
 samples/text/text_vc9.vcproj                     |  2 +-
 samples/thread/Makefile.in                       | 24 +++++++----------
 samples/thread/makefile.bcc                      |  2 +-
 samples/thread/makefile.gcc                      |  2 +-
 samples/thread/makefile.unx                      |  2 +-
 samples/thread/makefile.vc                       |  2 +-
 samples/thread/thread_vc7.vcproj                 |  2 +-
 samples/thread/thread_vc8.vcproj                 |  2 +-
 samples/thread/thread_vc9.vcproj                 |  2 +-
 samples/toolbar/Makefile.in                      | 24 +++++++----------
 samples/toolbar/makefile.bcc                     |  2 +-
 samples/toolbar/makefile.gcc                     |  2 +-
 samples/toolbar/makefile.unx                     |  2 +-
 samples/toolbar/makefile.vc                      |  2 +-
 samples/toolbar/toolbar_vc7.vcproj               |  2 +-
 samples/toolbar/toolbar_vc8.vcproj               |  2 +-
 samples/toolbar/toolbar_vc9.vcproj               |  2 +-
 samples/treectrl/Makefile.in                     | 24 +++++++----------
 samples/treectrl/makefile.bcc                    |  2 +-
 samples/treectrl/makefile.gcc                    |  2 +-
 samples/treectrl/makefile.unx                    |  2 +-
 samples/treectrl/makefile.vc                     |  2 +-
 samples/treectrl/treectrl_vc7.vcproj             |  2 +-
 samples/treectrl/treectrl_vc8.vcproj             |  2 +-
 samples/treectrl/treectrl_vc9.vcproj             |  2 +-
 samples/treelist/Makefile.in                     | 24 +++++++----------
 samples/treelist/makefile.bcc                    |  2 +-
 samples/treelist/makefile.gcc                    |  2 +-
 samples/treelist/makefile.unx                    |  2 +-
 samples/treelist/makefile.vc                     |  2 +-
 samples/treelist/treelist_vc7.vcproj             |  2 +-
 samples/treelist/treelist_vc8.vcproj             |  2 +-
 samples/treelist/treelist_vc9.vcproj             |  2 +-
 samples/typetest/Makefile.in                     | 24 +++++++----------
 samples/typetest/makefile.bcc                    |  2 +-
 samples/typetest/makefile.gcc                    |  2 +-
 samples/typetest/makefile.unx                    |  2 +-
 samples/typetest/makefile.vc                     |  2 +-
 samples/typetest/typetest_vc7.vcproj             |  2 +-
 samples/typetest/typetest_vc8.vcproj             |  2 +-
 samples/typetest/typetest_vc9.vcproj             |  2 +-
 samples/uiaction/Makefile.in                     | 24 +++++++----------
 samples/uiaction/makefile.bcc                    |  2 +-
 samples/uiaction/makefile.gcc                    |  2 +-
 samples/uiaction/makefile.unx                    |  2 +-
 samples/uiaction/makefile.vc                     |  2 +-
 samples/uiaction/uiaction_vc7.vcproj             |  2 +-
 samples/uiaction/uiaction_vc8.vcproj             |  2 +-
 samples/uiaction/uiaction_vc9.vcproj             |  2 +-
 samples/validate/Makefile.in                     | 24 +++++++----------
 samples/validate/makefile.bcc                    |  2 +-
 samples/validate/makefile.gcc                    |  2 +-
 samples/validate/makefile.unx                    |  2 +-
 samples/validate/makefile.vc                     |  2 +-
 samples/validate/validate_vc7.vcproj             |  2 +-
 samples/validate/validate_vc8.vcproj             |  2 +-
 samples/validate/validate_vc9.vcproj             |  2 +-
 samples/vscroll/Makefile.in                      | 24 +++++++----------
 samples/vscroll/makefile.bcc                     |  2 +-
 samples/vscroll/makefile.gcc                     |  2 +-
 samples/vscroll/makefile.unx                     |  2 +-
 samples/vscroll/makefile.vc                      |  2 +-
 samples/vscroll/vscroll_vc7.vcproj               |  2 +-
 samples/vscroll/vscroll_vc8.vcproj               |  2 +-
 samples/vscroll/vscroll_vc9.vcproj               |  2 +-
 samples/webview/Makefile.in                      | 24 +++++++----------
 samples/webview/makefile.bcc                     |  2 +-
 samples/webview/makefile.gcc                     |  2 +-
 samples/webview/makefile.unx                     |  2 +-
 samples/webview/makefile.vc                      |  2 +-
 samples/webview/webview_vc7.vcproj               |  2 +-
 samples/webview/webview_vc8.vcproj               |  2 +-
 samples/webview/webview_vc9.vcproj               |  2 +-
 samples/widgets/Makefile.in                      | 24 +++++++----------
 samples/widgets/makefile.bcc                     |  2 +-
 samples/widgets/makefile.gcc                     |  2 +-
 samples/widgets/makefile.unx                     |  2 +-
 samples/widgets/makefile.vc                      |  2 +-
 samples/widgets/widgets_vc7.vcproj               |  2 +-
 samples/widgets/widgets_vc8.vcproj               |  2 +-
 samples/widgets/widgets_vc9.vcproj               |  2 +-
 samples/wizard/Makefile.in                       | 24 +++++++----------
 samples/wizard/makefile.bcc                      |  2 +-
 samples/wizard/makefile.gcc                      |  2 +-
 samples/wizard/makefile.unx                      |  2 +-
 samples/wizard/makefile.vc                       |  2 +-
 samples/wizard/wizard_vc7.vcproj                 |  2 +-
 samples/wizard/wizard_vc8.vcproj                 |  2 +-
 samples/wizard/wizard_vc9.vcproj                 |  2 +-
 samples/wrapsizer/Makefile.in                    | 24 +++++++----------
 samples/wrapsizer/makefile.bcc                   |  2 +-
 samples/wrapsizer/makefile.gcc                   |  2 +-
 samples/wrapsizer/makefile.unx                   |  2 +-
 samples/wrapsizer/makefile.vc                    |  2 +-
 samples/wrapsizer/wrapsizer_vc7.vcproj           |  2 +-
 samples/wrapsizer/wrapsizer_vc8.vcproj           |  2 +-
 samples/wrapsizer/wrapsizer_vc9.vcproj           |  2 +-
 samples/xrc/Makefile.in                          | 24 +++++++----------
 samples/xrc/makefile.bcc                         |  2 +-
 samples/xrc/makefile.gcc                         |  2 +-
 samples/xrc/makefile.unx                         |  2 +-
 samples/xrc/makefile.vc                          |  2 +-
 samples/xrc/xrcdemo_vc7.vcproj                   |  2 +-
 samples/xrc/xrcdemo_vc8.vcproj                   |  2 +-
 samples/xrc/xrcdemo_vc9.vcproj                   |  2 +-
 samples/xti/Makefile.in                          | 24 +++++++----------
 samples/xti/makefile.bcc                         |  2 +-
 samples/xti/makefile.gcc                         |  2 +-
 samples/xti/makefile.unx                         |  2 +-
 samples/xti/makefile.vc                          |  2 +-
 samples/xti/xti_vc7.vcproj                       |  2 +-
 samples/xti/xti_vc8.vcproj                       |  2 +-
 samples/xti/xti_vc9.vcproj                       |  2 +-
 tests/Makefile.in                                | 34 +++++++-----------------
 tests/benchmarks/Makefile.in                     | 18 +------------
 tests/benchmarks/bench_vc7.sln                   |  1 -
 tests/benchmarks/bench_vc7_bench.vcproj          |  2 +-
 tests/benchmarks/bench_vc7_bench_graphics.vcproj |  2 +-
 tests/benchmarks/bench_vc7_bench_gui.vcproj      |  2 +-
 tests/benchmarks/bench_vc8.sln                   |  1 -
 tests/benchmarks/bench_vc8_bench.vcproj          |  2 +-
 tests/benchmarks/bench_vc8_bench_graphics.vcproj |  2 +-
 tests/benchmarks/bench_vc8_bench_gui.vcproj      |  2 +-
 tests/benchmarks/bench_vc9.sln                   |  1 -
 tests/benchmarks/bench_vc9_bench.vcproj          |  2 +-
 tests/benchmarks/bench_vc9_bench_graphics.vcproj |  2 +-
 tests/benchmarks/bench_vc9_bench_gui.vcproj      |  2 +-
 tests/benchmarks/makefile.bcc                    |  2 +-
 tests/benchmarks/makefile.gcc                    |  2 +-
 tests/benchmarks/makefile.vc                     |  2 +-
 tests/makefile.bcc                               |  2 +-
 tests/makefile.gcc                               |  2 +-
 tests/makefile.vc                                |  2 +-
 tests/test_vc7_test.vcproj                       |  2 +-
 tests/test_vc7_test_drawing.vcproj               |  2 +-
 tests/test_vc7_test_drawingplugin.vcproj         |  2 +-
 tests/test_vc7_test_gui.vcproj                   |  2 +-
 tests/test_vc8_test.vcproj                       |  2 +-
 tests/test_vc8_test_drawing.vcproj               |  2 +-
 tests/test_vc8_test_drawingplugin.vcproj         |  2 +-
 tests/test_vc8_test_gui.vcproj                   |  2 +-
 tests/test_vc9_test.vcproj                       |  2 +-
 tests/test_vc9_test_drawing.vcproj               |  2 +-
 tests/test_vc9_test_drawingplugin.vcproj         |  2 +-
 tests/test_vc9_test_gui.vcproj                   |  2 +-
 utils/Makefile.in                                |  2 +-
 utils/emulator/src/Makefile.in                   | 24 +++++++----------
 utils/emulator/src/emulator_vc7.vcproj           |  2 +-
 utils/emulator/src/emulator_vc8.vcproj           |  2 +-
 utils/emulator/src/emulator_vc9.vcproj           |  2 +-
 utils/emulator/src/makefile.bcc                  |  2 +-
 utils/emulator/src/makefile.gcc                  |  2 +-
 utils/emulator/src/makefile.vc                   |  2 +-
 utils/execmon/Makefile.in                        |  8 +-----
 utils/execmon/execmon_vc7.sln                    |  1 -
 utils/execmon/execmon_vc7.vcproj                 |  2 +-
 utils/execmon/execmon_vc8.sln                    |  1 -
 utils/execmon/execmon_vc8.vcproj                 |  2 +-
 utils/execmon/execmon_vc9.sln                    |  1 -
 utils/execmon/execmon_vc9.vcproj                 |  2 +-
 utils/execmon/makefile.bcc                       |  2 +-
 utils/execmon/makefile.gcc                       |  2 +-
 utils/execmon/makefile.vc                        |  2 +-
 utils/helpview/src/Makefile.in                   | 24 +++++++----------
 utils/helpview/src/helpview_vc7.vcproj           |  2 +-
 utils/helpview/src/helpview_vc8.vcproj           |  2 +-
 utils/helpview/src/helpview_vc9.vcproj           |  2 +-
 utils/helpview/src/makefile.bcc                  |  2 +-
 utils/helpview/src/makefile.gcc                  |  2 +-
 utils/helpview/src/makefile.vc                   |  2 +-
 utils/hhp2cached/Makefile.in                     | 24 +++++++----------
 utils/hhp2cached/hhp2cached_vc7.vcproj           |  2 +-
 utils/hhp2cached/hhp2cached_vc8.vcproj           |  2 +-
 utils/hhp2cached/hhp2cached_vc9.vcproj           |  2 +-
 utils/hhp2cached/makefile.bcc                    |  2 +-
 utils/hhp2cached/makefile.gcc                    |  2 +-
 utils/hhp2cached/makefile.vc                     |  2 +-
 utils/ifacecheck/src/Makefile.in                 |  8 +-----
 utils/ifacecheck/src/ifacecheck_vc7.sln          |  1 -
 utils/ifacecheck/src/ifacecheck_vc7.vcproj       |  2 +-
 utils/ifacecheck/src/ifacecheck_vc8.sln          |  1 -
 utils/ifacecheck/src/ifacecheck_vc8.vcproj       |  2 +-
 utils/ifacecheck/src/ifacecheck_vc9.sln          |  1 -
 utils/ifacecheck/src/ifacecheck_vc9.vcproj       |  2 +-
 utils/ifacecheck/src/makefile.bcc                |  2 +-
 utils/ifacecheck/src/makefile.gcc                |  2 +-
 utils/ifacecheck/src/makefile.vc                 |  2 +-
 utils/makefile.bcc                               |  2 +-
 utils/makefile.gcc                               |  2 +-
 utils/makefile.vc                                |  2 +-
 utils/screenshotgen/src/Makefile.in              | 24 +++++++----------
 utils/screenshotgen/src/makefile.bcc             |  2 +-
 utils/screenshotgen/src/makefile.gcc             |  2 +-
 utils/screenshotgen/src/makefile.vc              |  2 +-
 utils/screenshotgen/src/screenshotgen_vc7.sln    |  1 -
 utils/screenshotgen/src/screenshotgen_vc7.vcproj |  2 +-
 utils/screenshotgen/src/screenshotgen_vc8.sln    |  1 -
 utils/screenshotgen/src/screenshotgen_vc8.vcproj |  2 +-
 utils/screenshotgen/src/screenshotgen_vc9.sln    |  1 -
 utils/screenshotgen/src/screenshotgen_vc9.vcproj |  2 +-
 utils/wxrc/Makefile.in                           |  8 +-----
 utils/wxrc/makefile.bcc                          |  2 +-
 utils/wxrc/makefile.gcc                          |  2 +-
 utils/wxrc/makefile.vc                           |  2 +-
 utils/wxrc/wxrc_vc7.vcproj                       |  2 +-
 utils/wxrc/wxrc_vc8.vcproj                       |  2 +-
 utils/wxrc/wxrc_vc9.vcproj                       |  2 +-
 1031 files changed, 1846 insertions(+), 2624 deletions(-)

diff --git a/build/aclocal/bakefile.m4 b/build/aclocal/bakefile.m4
index 68773b6c66c..efa98ae0d07 100644
--- a/build/aclocal/bakefile.m4
+++ b/build/aclocal/bakefile.m4
@@ -598,16 +598,9 @@ AC_DEFUN([AC_BAKEFILE_RES_COMPILERS],
             dnl Check for win32 resources compiler:
             AC_CHECK_TOOL(WINDRES, windres)
          ;;
-
-      *-*-darwin* | powerpc-apple-macos* )
-            AC_CHECK_PROG(REZ, Rez, Rez, /Developer/Tools/Rez)
-            AC_CHECK_PROG(SETFILE, SetFile, SetFile, /Developer/Tools/SetFile)
-        ;;
     esac
 
     AC_SUBST(WINDRES)
-    AC_SUBST(REZ)
-    AC_SUBST(SETFILE)
 ])
 
 dnl ---------------------------------------------------------------------------
@@ -752,7 +745,7 @@ AC_DEFUN([AC_BAKEFILE],
     AC_SUBST(OBJCXXFLAGS)
 
 
-    BAKEFILE_BAKEFILE_M4_VERSION="0.2.9"
+    BAKEFILE_BAKEFILE_M4_VERSION="0.2.11"
 
     dnl includes autoconf_inc.m4:
     $1
diff --git a/build/bakefiles/config.bkl b/build/bakefiles/config.bkl
index 69b3da61f96..174a3dbeab4 100644
--- a/build/bakefiles/config.bkl
+++ b/build/bakefiles/config.bkl
@@ -397,7 +397,6 @@ to run the tests, include CppUnit library here.
     <!-- ================================================================== -->
 
     <if cond="FORMAT=='autoconf'">
-        <option name="DEREZ"/>
         <option name="TOOLKIT"/>
         <option name="TOOLKIT_LOWERCASE"/>
         <option name="TOOLKIT_VERSION"/>
diff --git a/build/bakefiles/wx.bkl b/build/bakefiles/wx.bkl
index 123be3faf22..9c7b419b401 100644
--- a/build/bakefiles/wx.bkl
+++ b/build/bakefiles/wx.bkl
@@ -150,29 +150,6 @@
             <linguas>it</linguas>
             <install-to>$(LOCALEDIR)</install-to>
         </gettext-catalogs>
-
-
-        <!-- Mac OS X resources: -->
-        <set var="MACOS_R">lib/libwx_$(TOOLCHAIN_NAME).$(WXSOVERSION[0]).r</set>
-        <set var="MACOS_RSRC">lib/libwx_$(TOOLCHAIN_NAME).$(WXSOVERSION[0]).rsrc</set>
-        <action id="cocoa-res" cond="TOOLKIT=='COCOA'">
-            <dependency-of>all</dependency-of>
-            <set var="__targetname">$(MACOS_R)</set>
-            <command>
-                $(REZ) -d __UNIX__ -useDF $(top_srcdir)/src/cocoa/dummy.r -o $(MACOS_RSRC)
-                $(DEREZ) $(MACOS_RSRC) -useDF >$(MACOS_R)
-            </command>
-            <clean-files>$(MACOS_RSRC) $(MACOS_R)</clean-files>
-        </action>
-        <action id="cocoa-res-install" cond="TOOLKIT=='COCOA'">
-            <dependency-of>install</dependency-of>
-            <depends>cocoa-res</depends>
-            <command>
-                $(INSTALL_DATA) $(MACOS_R) $(LIBDIR)
-                $(INSTALL_DATA) $(MACOS_RSRC) $(LIBDIR)
-            </command>
-        </action>
-
     </if>
 
 
